satellites.js

```diff
@@ -74,6 +74,24 @@ module.exports = function (RED) {
 
 		this.on('input', function (msg) {
 			var satellites = [];
+        // override tle1 if passed suitable value
+            if  ('satid' in msg)  {
```

Comment:
generally the Node-RED way is to only allow overrides if not set in the config... so check if satid is blank first then override.

Reply:
sorry I should have been more explicit -check if node.satid is not ""  then override. etc